docs/presentation/Content/ContentInstancesOfSpecificClasses.md

```diff
@@ -26,14 +27,28 @@ A specification that creates content for  instances of specific ECClasses.
 
 ### Attribute: `classes`
 
-Defines a single or an array of [multi schema classes](../Common-Rules/MultiSchemaClassesSpecification.md) which specify what and how the classes need to be selected to form result content.
+Defines a set of [multi schema classes](../Common-Rules/MultiSchemaClassesSpecification.md) that specify which ECClasses need to be selected to form the result.
 
 ```ts
 [[include:ContentInstancesOfSpecificClasses.Classes.Ruleset]]
 ```
 
 ![Example of using "classes" attribute](./media/contentinstancesofspecificclasses-with-classes.png)
 
+### Attribute: `excludedClasses`
+
+Defines a set of [multi schema classes](../Common-Rules/MultiSchemaClassesSpecification.md) that specify which ECClasses need to be excluded from the result.
```

Comment:
I think it's worth mentioning that this is expected to define a subset of what's in `classes`.

```suggestion
Defines a set of [multi schema classes](../Common-Rules/MultiSchemaClassesSpecification.md) that specify which ECClasses need to be excluded from the result.

Since the classes included in the result are defined using the [`classes` attribute](#attribute-classes), it's expected that if defines a superset of `excludedClasses`. If not, `excludedClasses` has no effect.
```

Reply:
I still feel like it would be worth adding a statement about the need for `excludedClasses` to derive from `classes` to have effect. @roluk, what do you think?

Reply:
No need to be this specific, unless specifying unrelated classes exhibits some unexpected effect. I left my suggestion in another comment thread.

Reply:
Thanks. The suggestion looks good.